music_app/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
          FlutterApplication and put your custom class here. -->
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:label="music_app"
+        android:label="Music_App"
```

Comment:
Also change the Main Project name as Music_App

Reply:
Also add ur mockup in the main readme file here

[ReadMe Link](https://github.com/Safnaj/Flutter-UI-Samples/blob/master/README.md)

Reply:
ok. got it